packages/wrangler/src/dev.tsx

```diff
@@ -227,6 +231,12 @@ function useLocalWorker(props: {
         return;
       }
 
+      if (props.public) {
+        console.error(
+          '⎔ A "public" folder is not yet supported in local mode.'
+        );
```

Comment:
I think this should throw an error.
If the developer expects there to be a public asset directory available then it doesn't make much sense to try to run it locally without.

Reply:
Good point. I'll file an issue as well.

Reply:
We'll just use https://github.com/cloudflare/wrangler2/issues/268 as the issue when we get to it